foundation/application_test.go

```diff
@@ -311,9 +312,14 @@ func (s *ApplicationTestSuite) TestMakeRateLimiter() {
 	s.NotNil(s.app.MakeRateLimiter())
 }
 
+type Engine struct {
+	contractsroute.Engine
```

Comment:
> I want to rename `route.Engine` to `route.Route`, and `route.Route` to `route.Router`, I think It will be clearer, if you have no problem, I'll do it in another PR and modify the `goravel/gin` repository at the same time.

no problem